frontend/src/pages/Agency/Agency.js

```diff
@@ -224,6 +175,51 @@ function AgencyTable() {
     return false;
   }
 
+  /**
```

Comment:
Why was this function moved down in the file?

Reply:
More linting - eslint wanted functions to be defined in different orders in a few files (I think to make sure they are defined before used)